components/script/dom/htmlfontelement.rs

```diff
@@ -174,8 +174,8 @@ fn parse_size(mut input: &str) -> AttrValue {
 
     // Step 9
     match parse_mode {
-        ParseMode::RelativePlus => value = 3 + value,
-        ParseMode::RelativeMinus => value = 3 - value,
+        ParseMode::RelativePlus => value += 3,
+        ParseMode::RelativeMinus => value -= 3,
```

Comment:
Since subtraction is not commutative, this change is not equivalent to the original code i.e `value -= 3` expands to `value = value - 3` whereas we want `value = 3 - value`

I think we can simply revert the change on line 178 alone.

Reply:
I have reverted the changes, it should be fine now.